environment.yml

```diff
@@ -21,8 +21,7 @@ dependencies:
 
   # core dependencies
   - django>=3.2,<6
-  - channels
-  - daphne
+  - channels["daphne"]
```

Comment:
Is this valid syntax for a conda package? I though that was a `pip` thing.

Reply:
Hm... It must be, since I installed tethys fresh from this branch on a new machine a couple of days ago and it worked without issue.